src/components/ChatEntry.js

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
 
 const ChatEntry = (props) => {
+
+  const toggleLikeStatus = () => {
+    // build out new (updated) entry
+    const updatedMessage = {
```

Comment:
There's no advantage to enclosing and passing the entire message object in this way, so I'd suggest a different approach. From a maintainability standpoint, it's less than ideal since if any changes to the object shape occur, you'd have to reflect them in this object, which is itself just a consumer of data, so it couples the shape of this object very tightly to a component that doesn't need to know anything about said data.